config/connectSlack.js

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable no-console */
+const { App } = require("@slack/bolt");
+
+const modalTemplate = require("./modal.json");
+
+const app = new App({
+  signingSecret: process.env.SLACK_SIGNING_SECRET,
+  token: process.env.SLACK_BOT_TOKEN,
+  appToken: process.env.SLACK_APP_TOKEN,
+  socketMode: true,
+});
+
+(async () => {
+  await app.start(process.env.PORT || 3000);
+
+  console.log("⚡️ Bolt app is running!");
+})();
+
+(function () {
+  const channelList = {};
+
+  app.command("/code", async ({ ack, body, client, logger }) => {
+    const { channel_id } = body;
```

Comment:
카멜케이스로 변환해서 사용하는 편이 통일성이 있을 것 같아요.

Reply:
다음과 같이 작성하면 될 것 같네요!

```suggestion
    const { channel_id: channelId } = body;
```